analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -1394,7 +1394,7 @@ public static void OrConstraint()
                 | helper == Helper.C | helper == Helper.D | helper == Helper.E | helper == Helper.F
                 | helper == Helper.G | helper == Helper.H | helper == Helper.I | helper == Helper.J)
             {
-                o.ToString(); // Noncompliant (old engine: FN, the condition state generation is too big to explore all constraint combinations)
+                o.ToString(); // FN (old engine: FN, the condition state generation is too big to explore all constraint combinations)
```

Comment:
Why does the PR break these taste cases?

Reply:
There's something wrong about how the enum symbol is handled. Currently, its somehow handled and tracked as a symbol, while it should be handled as a const value. I will need to investigate further.